lhotse/kaldi.py

```diff
@@ -292,8 +296,27 @@ def make_wavscp_string(source: AudioSource, sampling_rate: int) -> str:
         return f"{source.source} |"
     elif source.type == "file":
         if Path(source.source).suffix == ".wav":
-            return source.source
+            audios = dict()
```

Comment:
Yeah that looks good to me! Can you change the return type of `make_wavscp_string` to `Dict[int, str]` (and maybe rename to sth like `make_channel_to_wavscp_string_map` or less verbose if you have a better idea).